patches/chromium/fix_restore_original_resize_performance_on_macos.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <marshallofsound@electronjs.org>
+Date: Tue, 21 Nov 2023 15:03:21 -0800
+Subject: fix: restore original resize performance on macOS
+
+Reverts https://chromium-review.googlesource.com/c/chromium/src/+/3118293 which
+fixed a android only regression but in the process regressed electron window
+resize perf wildly on macOS.
```

Comment:
Per our [patch policy](https://github.com/electron/electron/blob/main/docs/development/patches.md#patch-justification), all patches are required to include in their description either (a) conditions for the patch's removal, or (b) a reason why the patch can't be removed. Please update this description 🙇🏻